internal/constants/const.go

```diff
@@ -15,4 +15,7 @@ const (
 	FieldOwner = "package-operator"
 	// OwnerStrategyAnnotationKey is the k8s annotation key that denotes the owner of a resource.
 	OwnerStrategyAnnotationKey = "package-operator.run/owners"
+
+	// Verbosity level for Info logs that are hidden by default.
+	LogLevelInfo = 1
```

Comment:
This name is misleading, logs with `log.V(1)` don't actually show up on info level (`LOG_LEVEL=0`), they show up one level higher, so this should be called something like `LogLevelDebug`.